modules/composer-ffmpeg/src/main/java/org/opencastproject/composer/impl/ComposerServiceImpl.java

```diff
@@ -1321,9 +1346,10 @@ protected String process(Job job) throws ServiceRegistryException {
           serialized = MediaPackageElementParser.getArrayAsXml(resultingElements);
           break;
         case ImageConversion:
+          String[] encodingProfilesArr = StringUtils.split(arguments.get(0), ',');
```

Comment:
Just to be safe: Can we properly serialize this (e.g. use Gson)? This should usually not break but why knows when someone starts using `,` in an encoding profile.